cmd/helm/resolver/resolver.go

```diff
@@ -55,6 +57,23 @@ func (r *Resolver) Resolve(reqs *chartutil.Requirements, repoNames map[string]st
 	locked := make([]*chartutil.Dependency, len(reqs.Dependencies))
 	missing := []string{}
 	for i, d := range reqs.Dependencies {
+		if strings.HasPrefix(d.Repository, "file://") {
+			depPath, err := filepath.Abs(d.Repository[7:])
```

Comment:
Instead of using the slice notation here, I'd recommend `strings.TrimPrefix(d.Repository, "file://")`. That will make it clearer and not use a magic number